ratings.toml

```diff
@@ -28,19 +28,30 @@
         { MaxThreshold = 10000000,ChancePercent = 24},
     ]
 
-[ShardChain.RatingSteps]
-    HoursToMaxRatingFromStartRating = 72
-    ProposerValidatorImportance = 1.0
-    ProposerDecreaseFactor = -4.0
-    ValidatorDecreaseFactor = -4.0
-    ConsecutiveMissedBlocksPenalty = 1.10
+[ShardChain]
+    [[ShardChain.RatingStepsByEpoch]]
+        EnableEpoch = 0
+        HoursToMaxRatingFromStartRating = 72
+        ProposerValidatorImportance = 1.0
+        ProposerDecreaseFactor = -4.0
+        ValidatorDecreaseFactor = -4.0
+        ConsecutiveMissedBlocksPenalty = 1.10
+    [[ShardChain.RatingStepsByEpoch]]
+        EnableEpoch = 1763
```

Comment:
Everything like on devnet & testnet, with the exception of `EnableEpoch` :heavy_check_mark: